arlo_server/audit_boards.py

```diff
@@ -119,3 +122,66 @@ def create_audit_boards(election: Election, jurisdiction: Jurisdiction, round_id
     assign_sampled_ballots(jurisdiction, round, audit_boards)
 
     return jsonify(status="ok")
+
+
+def round_status_by_audit_board(audit_boards: List[AuditBoard]) -> Dict[str, JSONDict]:
+    if len(audit_boards) == 0:
+        return {}
+
```

Comment:
worth a check that all audit boards have the same `jurisdiction_id` to make sure we don't get into nonsensical results? I guess the `get()` will currently return `None`, so it's not a huge problem, but I think our future selves will enjoy an error message on feeding an improper list of audit boards.

Reply:
put another way, I'm wondering if what you want is `round_status_by_jurisdiction`, since no matter how many audit boards you're feeding this function, it's going to do roughly the same amount of work overall.

Reply:
Can you share more about what that would be guarding against? This function gets called with the result of a db query that filters by jurisdiction. I originally had the signature `round_status_by_audit_board(jurisdiction_id, round_id, audit_boards)` but took out the first two args to make it more concise.

Reply:
It *should* be called with the result of that query, but what if, at some point, it's called with a list of audit boards that come out of a poorly formed query, or I look at that signature and think somehow I can query all audit boards across jurisdictions?

I'm suggesting that a better sig might be `(jurisdiction_id, round_id)`

Reply:
How's this look? 3ab03f4

Reply:
nice :)